config/environments/development.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
-
+  OpenSSL::SSL.send(:remove_const, :VERIFY_PEER)
```

Comment:
Is this the only way to disable SSL peer verification? Is that what this is doing?